addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -191,6 +192,7 @@ const Wysiwyg = Widget.extend({
             categories: powerboxOptions.categories,
             plugins: options.editorPlugins,
             direction: localization.direction || 'ltr',
+            collaborationClientAvatar: `${browser.location.origin}/web/image?model=res.users&field=avatar_128&id=${this.getSession().uid}`,
```

Comment:
Wouldn't it work with a relative url instead of using browser location origin ?

Reply:
It might actually, I honestly just copied this from another place where the avatar was fetched :sweat_smile: Want me to try it? Is it worth it? I feel this is safer.

Reply:
What is the other place ? Care to share your sources ? :p

Reply:
`addons/web/static/src/webclient/user_menu/user_menu.js`:

```js
setup() {
    this.user = useService("user");
    const { origin } = browser.location;
    const { userId } = this.user;
    this.source = `${origin}/web/image?model=res.users&field=avatar_128&id=${userId}`;
}
```